src/otx/core/model/module/base.py

```diff
@@ -178,8 +175,8 @@ def load_state_dict(self, ckpt: dict[str, Any], *args, **kwargs) -> None:
                 f"Data classes from training data: {self.meta_info.label_names}",
             )
             self.register_load_state_dict_pre_hook(
-                self.meta_info.label_names,
-                ckpt_meta_info.class_names,
+                self.meta_info.class_names,
```

Comment:
```suggestion
                self.meta_info.label_names,
```
As far as I know, both `self.meta_info` and `ckpt_meta_info` is `LabelInfo` class. shouldn't both of them access to `label_names`?

Reply:
That looks like a merge artifact, I'll revert this